content/docs/how-we-work/baseline/git-workflow.md

```diff
@@ -27,9 +27,20 @@ The branches can be deleted immediately and [automatically](https://docs.github.
 
 ### Protecting the default branch on GitHub
 
-[GitHub provides a variety of protection mechanism](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches) for different branches. It especially recommends that the main branch (often called `main` or `master`) is protected from accidental pushes.
+GitHub provides a variety of protection mechanism for different branches.
 
-We recommend that the default branch is always protected. At a minimum, a PR should be required before merging to the main branch. You can also enforce conditions such as the PR must have approvals and requiring reviews from code owners. More rules can be found in the [documentation](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches).
+Below are our recommended branch protection settings.
+
+* trunk - main/master:
+  * [Require pull request reviews before merging](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches#require-pull-request-reviews-before-merging)
+  * [Require status checks before merging](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches#require-status-checks-before-merging)
+  * [Restrict who can push to matching branches](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches#restrict-who-can-push-to-matching-branches) which is usually restricted to Admin so they can push from CLI if needed in some cases.
+  * [Require conversation resolution before merging](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches#require-conversation-resolution-before-merging) (Optional)
+  * [Do not allow bypassing the above settings](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches#do-not-allow-bypassing-the-above-settings) (Optional)
```

Comment:
I'd rather not mention this. Sometimes, bypassing is required and having this for admins is beneficial (when you want to push an urgent fix and no one is available to review). I know you have marked it as optional but as this is a checkbox, people will think that the recommendation is easy to implement and do it anyway.

Reply:
Ok Removing this